### ISSUE TITLE
en passant target square not populated when pawn moves 2 sqares

### ISSUE THREAD

**Issue by sdxsharp** [niklasf/python-chess#1082]
```py
import chess

print(chess.__version__)
board = chess.Board()
board.set_fen('rnbqkbnr/pppppppp/8/8/8/8/PPPPPPPP/RNBQKBNR w KQkq - 0 1')
board.push_san('e4')
print(board.fen())
```

output:

1.10.0
rnbqkbnr/pppppppp/8/8/4P3/8/PPP1PPPP/RNBQKBNR b KQkq - 0 1

the en passant sqare is not populated!!!

output should be:

rnbqkbnr/pppppppp/8/8/4P3/8/PPPP1PPP/RNBQKBNR b KQkq e3 0 1

**Comment by MarkZH** [comment]
By default, the en passant square is only listed if there is a legal en passant move. If you want the usual FEN style, use `print(board.fen(en_passant='fen'))`.

[Board.fen() method parameter documentation](https://github.com/niklasf/python-chess/blob/eaa6eb3b023e92441f9de810785bb1716dd4d0b5/chess/__init__.py#L2530-L2535)

**Comment by sdxsharp** [comment]
Thanks! I wish I could've found it myself in the docs, but for someone who never used python docs the search result looks like a hot mess (due to lack of class information and hard to parse optional arguments or even identify what is going on) and the navigation is also not super obvious (not sure why I didn't look in Core) (and even if I'd found the right class the font size for the actual functions is tiny af).
